src/components/accessories/patientNewBill/itemsTable/BillItemsTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { FC } from "react";
+import Table from "../../table/Table";
+import { header, mockData, order } from "./consts";
+import { currencyFormat } from "../../../../libraries/formatUtils/currencyFormatting";
+import { useTranslation } from "react-i18next";
+
+interface IOwnProps {
+  handleEdit: (row: any) => void;
+  handleDelete: (row: any) => void;
+}
+
+const BillItemsTable: FC<IOwnProps> = ({ handleEdit, handleDelete }) => {
+  const { t } = useTranslation();
+  const label = {
+    type: t("bill.type"),
+    description: t("bill.description"),
+    qty: t("bill.quantity"),
+    cost: t("bill.amount"),
```

Comment:
I think that we should use "Total amount" or "Amount per item" based if the description is referred to the total amount (quantity * single_price) or the price of a single item.

Reply:
We prefer to use just amount here in order to save space. There is no much space to display a long column label.